fabric-enchantment-api-v1/src/main/resources/fabric-enchantment-api-v1.mixins.json

```diff
@@ -0,0 +1,16 @@
+{
+  "required": true,
+  "package": "net.fabricmc.fabric.mixin.enchantment",
+  "compatibilityLevel": "JAVA_8",
+  "mixins": [
+    "EnchantmentHelperMixin",
+    "EnchantedBookItemMixin",
+    "AnvilScreenHandlerMixin",
+    // "DisabledAnvilScreenHandlerMixin"
```

Comment:
Obviously this is disabled until we decide whether it is needed. Just noting here so we don't have an issue at release.